gradle.properties

```diff
@@ -33,7 +33,7 @@ stdlibLogVersion=2.11.0-20241122-150100-2e1a3ca
 stdlibOsVersion=1.9.0-20241122-071800-1fae486
 stdlibTaskVersion=2.6.0-20241122-124800-cb84357
 observeInternalVersion=1.4.0-20241122-185200-9e5b1bb
-stdlibDataXmldataVersion=1.1.0-20241122-195700-b63fee2
+stdlibDataXmldataVersion=1.2.0-20241129-110700-14aff6e
```

Comment:
We should run the build locally to update the dependencies.toml file. This will fail the build otherwise.

Reply:
Sorry, my mistake. Please ignore.